auth-server/README.md

```diff
@@ -29,6 +29,8 @@ To deploy the [`Auth server`](/auth-server/) manually:
 1. [Create](https://cloud.google.com/compute/docs/instances/create-start-instance) a Compute Engine VM instance with your preferred machine type and OS (for example, e2-medium and Debian 10), then [reserve a static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address)
 1. Verify that your VM instance has the `Stackdriver Logging API` write [access scope](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) (on by default) and that your VM’s [service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Logs Writer`](https://cloud.google.com/logging/docs/access-control) role (off by default) 
 1. Check out the latest code from the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies/)
+1. Update the organization name as desired for org.name in the [application.properties](oauth-scim-service/src/main/resources/application.properties) file
```

Comment:
This shouldn't be included in the README - the README already points the user to the applications.properties file where the comments of that file describe what to change. Please remove.

Reply:
Fixed review comment. Removed this point from `README` file and added comment in `application.properties`